Ui/DataProvider/Product/Form/Modifier/AssignVendor.php

```diff
@@ -51,27 +63,31 @@ public function modifyData(array $data)
      */
     public function modifyMeta(array $meta)
     {
-        $meta = array_replace_recursive(
-            $meta,
-            [
-                'assign_vendor' => [
-                    'arguments' => [
-                        'data' => [
-                            'config' => [
-                                'label' => __('Assign Product to Vendor'),
-                                'componentType' => Fieldset::NAME,
-                                'dataScope' => 'data.product.assign_vendor',
-                                'collapsible' => false,
-                                'sortOrder' => 5,
+        if (!($this->isModuleEnable())) {
```

Comment:
This condition checks if Webkul module is enable or not. 
If its enable we remove the "Assign Product to Vendor" dropdown.
@ynaxonBalance